### ISSUE TITLE
a11y improvements for Legal & policies and Terms

### ISSUE THREAD

**Issue by maria-jon** [openaq/openaq.org#1131]
Legal & policies and Terms accessibility can improve in a few places: 

## Legal & policies

- Certifications missing link and alt text  
- Select-elements missing labels (aria-labelledby heading?)

## Terms

- Multiple h1 tags, review all heading levels
